app/models/submission.rb

```diff
@@ -9,6 +9,9 @@ class Submission < ApplicationRecord
 
   scope :non_flagged, -> { where(flagged: false) }
 
+  paginates_per 10
```

Comment:
Sync with Natalie on this, but 100 at a time feels more reasonable than 10.

I'm thinking about if sorting submissions by `created_at DESC` would make more sense here than `ASC`. 
My thought is about having thousands of responses, and how to do I see latest easily? 
* look at first page of `DESC` sorted results
* click to last page and view `ASC` sorted results